### PR TITLE
Migrate from 1.6 structure/event to 1.7

### LINKED ISSUE #34: Update to sylius 1.7 ?
Hello,

This theme is still alive ? because no update or no new release since sylius 1.7

++++

### PR DESCRIPTION
**Pull request by pocky**
First pass:

- [x] global layout
- [x] Taxon
- [x] Register
- [x] ProductReview
- [x] Product
- [x] Order
- [x] Login
- [x] Homepage
- [x] Email
- [x] Contact
- [x] Chekout
- [x] Cart
- [x] Account

Second pass:

- [x] Add all class/id `sylius-*`
- [x] Add all sylius_test_html_attributes

Third pass:
- [x] Rewrite HTML
- [x] Double check `Common`
- [x] Click Click Click

Close #34

### CODE REVIEW

File: SyliusShopBundle/views/Product/Index/_pagination.html.twig [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {% set paginationLimits = (resources.definition.limits) %}
 
+<div class="d-flex justify-content-end my-3">
```
Comment (FlashPanther) [inline SyliusShopBundle/views/Product/Index/_pagination.html.twig:3]: Tag unclosed in this file ... 
You 'll get an error if you disable the pagination block into config

Should be moved to _main.html.twig ?

File: SyliusShopBundle/views/Product/Index/_sorting.html.twig [hunk 2]
```diff
@@ -46,3 +46,4 @@
 </div>
 
 {% endif %}
+</div>
```
Comment (FlashPanther) [inline SyliusShopBundle/views/Product/Index/_sorting.html.twig:49]: Related to previous comment

**Comment by pocky** [comment]
screencapture-localhost-en-US-2020-05-28-23_13_43

**Comment by antiseptikk** [comment]
It seems to be good for me. Tested with and without a custom theme based on it.

Thanks!

**Pull request merged**